database/oracle/startDB.sh

```
@@ -0,0 +1,8 @@
+#!/bin/bash
+echo "Starting Oracle Listener..."
+lsnrctl start
+
+echo "Starting Oracle Database..."
+sqlplus / as sysdba <<EOF
```

Shellcheck

[info] The mentioned syntax error was in this simple command.

(SC1009)